src/@next/Banner/Banner.tsx

```diff
@@ -84,9 +84,9 @@ export const Banner = React.forwardRef<HTMLDivElement, BannerProps>(
           {...props}
         >
           <StyledBannerTitleContainer>
-            <Icon name={iconNameValue} />
+            {showIcon && <Icon name={iconNameValue} />}
             {title && (
-              <StyledBannerTitle>
+              <StyledBannerTitle data-show-icon={showIcon}>
```

Comment:
it seems like you can apply this to StyledBanner itself since you're putting the condition on each of the children? can you check?

Reply:
If `showIcon` is `true`, I don't want to apply it here: `{showIcon && <Icon name={iconNameValue} />}`
That is one of the children, so I can't apply it to the parent div directly, otherwise I'll get something like this:
![image](https://github.com/glints-dev/glints-aries/assets/67429929/d6fdbb41-1ae9-41e8-b8cf-af0c5f97cba0)

Reply:
what I also meant was, based on Figma, it will always have the padding with or without the Icon. We should retain that on the parent but apply the spacing of the Icon to itself instead to isolate it from the other children.

I think the structure of the components should also be the same with the fixed banner:
```js
<StyledBanner>
  <Icon>
  <Content>
    {children}
    {actions}
  </Content>
  <CloseIcon>
</StyledBanner>
```

In this case, there'll be no overlapping of the Icon and the content and you'll be able to apply the exact spacing value defined.

Reply:
> it will always have the padding with or without the Icon

thanks, let me confirm with the designer first about this
![image](https://github.com/glints-dev/glints-aries/assets/67429929/65fbe636-5ba6-450b-bfcd-73d83db16ca3)
not sure if the screenshot here is intended or is it a mistake

Reply:
> always have the padding with or without the Icon

asked designer that is not intended
I'll just make a slight change to make the structure same as fixed banner as you mentioned